Libraries/pbxbuild/Sources/Build/Environment.cpp

```diff
@@ -52,7 +53,8 @@ Default(Filesystem const *filesystem)
      */
     specManager->registerDomains(filesystem, pbxspec::Manager::DefaultDomains(*developerRoot));
 
-    std::shared_ptr<xcsdk::SDK::Manager> sdkManager = xcsdk::SDK::Manager::Open(filesystem, *developerRoot);
+    auto config = std::unique_ptr<config::Config>(new config::Config(filesystem));
```

Comment:
Any reason to use `std::unique_ptr<Config>` here versus just leaving the `Config` on the stack?